src/IW/Db/Repo.hs

```diff
@@ -43,3 +44,18 @@ upsertRepos = executeMany [sql|
         descr = EXCLUDED.descr
       , categories = EXCLUDED.categories;
 |]
+
+-- | Update a repo's categories field.
+updateRepoCategories
+    :: ( WithDb env m
+       )
```

Comment:
When there is only a single constraint, no need to spawn multiple redundant lines. This can be just:

```haskell
    :: WithDb env m
    =>